docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,98 @@
+
+### Moderate development speed, moderate flexibility
+
+Products like airplane.dev, interval.com fall under this category. They have APIs for all admin use cases; while this can be great for those who enjoy writing code, it is a bit too much as adding a component also requires writing code. Their main advantage over previous category is that they store their configuration in local code files. This puts more control over the development lifecycle back in the developer's hands. Users can version control, deploy, and use any IDE they want.
+
+### Slow development speed, more flexibility
+
+Products like react admin, refine.dev fall in this category, they are all code and have APIs for everything to build an admin panel from scratch. They don't support cloud hosting and are a relatively cheaper alternative.
+
+We'll do a detailed comparison with these tools in the future.
+
+## Toolpad
+
+How is Toolpad different from the above tools, and why should you use it? Toolpad is a **flexible and fast** way to build a back office application. Let's see how:
+
+### Front end UI
+
+Toolpad works as a drag-and-drop UI builder and uses MUI's own Material UI components. Material UI has almost every component required for back-office use cases; currently, 16 of them are there in Toolpad but we are always adding more.
+
+For more components, Toolpad supports adding any number of custom components. You can create your own by writing React code.
+
+Drag-and-drop builder helps you assemble a quick UI and gives you a head start for the next steps.
+
+### Consistent Design
+
+Maintaining a consistent design language throughout a web application is crucial for a seamless user experience. Toolpad helps you achieve this by providing pre-built components that follow Material Design guidelines. This ensures your application looks consistent across all pages and screens.
+
+Toolpad also supports adding a theme to a page, it can help with a UI that matches your brand guidelines. The components have overrides in case wish to customize them further.
+
+### Connecting to data
+
+Imagine you've built the front end, the next step is connecting your data. In Toolpad, you can bring your entire data from your existing backend in two ways:
+
+- REST APIs can be easily linked to Toolpad through **HTTP queries**. This is the most common way of getting data to Toolpad
+- **Custom functions** where you can write code in your own IDE to connect to any database. If there is business logic in your customer-facing application that could be useful in Toolpad; you can simply import it into Toolpad. You can reuse your database models, client libraries, and secrets. However, remember that currently only Node.js backend is supported in Toolpad.
+
+Later we'll also support REST API like UI for graphQL, Open API, etc.
```

Comment:
I suggest not making commitments in docs pages that we might not remember to maintain. Instead, we can update this with the link to the public roadmap once that is ready.